pkg/controller/certapi/certificate.go

```diff
@@ -45,17 +44,8 @@ func (c *Controller) HandleCertificate() http.Handler {
 			return
 		}
 
-		// This is a non terminal error, as we're only using the realm for stats.
-		realm, err := authApp.Realm(c.db)
```

Comment:
We don't need to lookup the realm to get it's ID - it's already a field on the authapp, so dropped this db lookup.